spark-indexer/src/main/java/org/uniprot/store/spark/indexer/suggest/mapper/flatfile/FlatFileToCofactorChebi.java

```diff
@@ -0,0 +1,59 @@
+package org.uniprot.store.spark.indexer.suggest.mapper.flatfile;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+import org.apache.spark.api.java.function.PairFlatMapFunction;
+import org.uniprot.core.uniprot.comment.*;
+import org.uniprot.core.util.Utils;
+import org.uniprot.store.spark.indexer.suggest.SuggesterUtil;
+
+import scala.Tuple2;
+
+/**
```

Comment:
I think a short javadoc description of the mapping would be useful for newcomers.

Reply:
... and the same for lots of the mapping classes. Sorry!